src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsProvider.java

```diff
@@ -55,20 +56,33 @@
 import org.ow2.proactive.connector.iaas.model.ScriptResult;
 import org.ow2.proactive.connector.iaas.model.Tag;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 import com.google.common.collect.Lists;
 
+import lombok.Getter;
+import lombok.Setter;
+
 
 @Component
 public abstract class JCloudsProvider implements CloudProvider {
 
+    private static final String DEFAULT_CONNECTOR_IAAS_VM_USER_LOGIN = "admin";
+
+    private final Logger logger = Logger.getLogger(JCloudsProvider.class);
+
     @Autowired
     private JCloudsComputeServiceCache jCloudsComputeServiceCache;
 
     @Autowired
     private TagManager tagManager;
 
+    @Getter
+    @Setter
+    @Value("${connector-iaas.vm-user-login}")
```

Comment:
If I remember well you can provide the default value from this field with something like: `"${connector-iaas.vm-user-login:admin}"`

Reply:
ok, I will check that instead of using the .orElse of the Optional. Thanks